core/src/Kokkos_View.hpp

```diff
@@ -1422,8 +1429,12 @@ class View : public ViewTraits<DataType, Properties...> {
   KOKKOS_DEFAULTED_FUNCTION
   View() = default;
 
-  KOKKOS_DEFAULTED_FUNCTION
-  View(const View&) = default;
+  KOKKOS_INLINE_FUNCTION
+  View(const View& rhs) : m_map(rhs.m_map), m_track(*this, rhs) {}
+
+  KOKKOS_INLINE_FUNCTION
+  View(const View& rhs, bool _skip_hooks)
+      : m_map(rhs.m_map), m_track(*this, rhs, _skip_hooks) {}
```

Comment:
This seems like a really minor thing to be using up the cognitive space of the `View(View, bool)` constructor. I doubt most users would expect a constructor with the second argument being boolean has anything to do with hooks. You should do something like `ViewConstructSkipHooks` or something like that, consistent with other special constructors of `View`.

Reply:
this is private now 60b42a7d87599a9d2c2bfe913cb811b38a3b6f9f